### ISSUE TITLE
Add UUID as a part of SimpleEvent

### ISSUE THREAD

**Issue by kristinapathak** [xmidt-org/wrp-go#61]
Currently, [glaukos](https://github.com/xmidt-org/glaukos) is utilizing a [bug in Caduceus](https://github.com/xmidt-org/caduceus/blob/e088f9dbef3855c620470127cae70af57e5ffed4/http.go#L115
) in order to skip over an event in the event history that is the same event sent to the webhook.  The transaction UUID is meant to be no longer be set/created in Caduceus as a part of this Caduceus issue:
https://github.com/xmidt-org/caduceus/issues/174#issuecomment-665274785

A UUID is helpful for identifying events.  It would be nice to add it as a required field to SimpleEvent in some form, even if `TransactionUUID` doesn't make sense.

**Comment by schmidtw** [comment]
I agree, I think excluding the TransactionUUID was a mistake in the SimpleEvent.  We'll need to rev the spec to add this.

**Comment by kristinapathak** [comment]
This would implement the change in the spec described in https://github.com/xmidt-org/docs/issues/74.
